php/Terminus/Models/Workflow.php

```diff
@@ -81,6 +81,22 @@ public function isSuccessful() {
   }
 
   /**
+   * Detects if the workflow has Operation Logs
+   *
+   * @return [boolean] $has_logs True if worklow has logs
+   */
+  public function hasLogs() {
+    $operations = $this->operations();
+    $has_logs = false;
+    foreach ($operations as $operation) {
+      if ((boolean)$operation->get('log_output')) {
+        $has_logs = true;
+      }
```

Comment:
Oh yeah, it could do an early return from the loop. Whelp, I just rewrote it as an inject.